aubio-rs/src/filterbank.rs

```diff
@@ -0,0 +1,141 @@
+use crate::{
+    check_init,
+    Error::{InvalidArg},
+    ffi,
+    vec::{FVecMut, CVec, FMat, FMatVecs},
+    Result,
+    Status,
+};
+
+/**
+ * Filterbank object.
+ *
+ * General-purpose spectral filterbank object.
+ */
+pub struct FilterBank {
+    filterbank: *mut ffi::aubio_filterbank_t,
+    n_filters: usize,
+    win_s: usize,
```

Comment:
Because size checks is optional feature, this fields needs only when "check-size" feature enabled.
Add `#[cfg(feature = "check-size")]` before both fields.

See below.

Reply:
Don't we need them since they reflect the way `filterbank_t` looks like? (and it's also useful to know that directly from Rust)
Since we have no easy way to retrieve that using aubio helpers

Reply:
Of course we need them but only when "check-size" crate feature enabled.

Reply:
@katyo should be as you wanted it :)